byte.h

```diff
@@ -3,11 +3,11 @@
 
 #include <string.h>
 
-extern unsigned int byte_chr();
-extern unsigned int byte_rchr();
-extern void byte_copy();
-extern void byte_copyr();
-extern void byte_zero();
+extern unsigned int byte_chr(char *s, unsigned int n, int c);
+extern unsigned int byte_rchr(char *s, unsigned int n, int c);
+extern void byte_copy(char *to, unsigned int n, char *from);
+extern void byte_copyr(char *to, unsigned int n, char *from);
+extern void byte_zero(char *s, unsigned int n);
 
```

Comment:
http://git.skarnet.org/cgi-bin/cgit.cgi/skalibs/tree/src/include/skalibs/bytestr.h

This above provides example on how to convert DJB-style functions to <string.h> ones.

Reply:
Not in this PR ;)